### PR TITLE
[SMALLFIX] Fix test with -PhdfsTest 

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

**Comment by apc999** [comment]
Jenkins, test this please

File: underfs/hdfs/pom.xml [hunk 1]
```diff
@@ -288,11 +279,13 @@
         <dependency>
           <groupId>org.apache.hadoop</groupId>
           <artifactId>hadoop-common</artifactId>
+          <version>${hadoop.version}</version>
```
Comment (calvinjia) [inline underfs/hdfs/pom.xml]: Could you comment somewhere why we do not use dependency management for this?

**Comment by calvinjia** [comment]
Jenkins, test this please.

File: underfs/hdfs/pom.xml (hunk 1 above)
Comment (apc999) [inline underfs/hdfs/pom.xml]: Added the explanation to parent pom

**Pull request merged**